README.md

```diff
@@ -1,3 +1,129 @@
+
+***
+### 2. SCSS
+1. Copy the '@momentum-ui/icons/fonts' directory to the 'fonts' directory for your site.
+2. Add an '$brand-font-folder, $icon-font-path, and $images-path' variables to your app's variables.scss file.
+
+    ```scss
+    $brand-font-folder: 'path/to/fonts/directory';
+    $icon-font-path: 'path/to/fonts/directory';
+    $images-path: 'path/to/images/directory';
+    ```
+
+3. Import '@momentum-ui/core/scss/momentum-ui.scss' into your main entry Scss file AFTER the variables.
+
+    `@import '~@momentum-ui/core/scss/momentum-ui';`
+
+4. Compile your Scss using your static compiler or bundler (Webpack, Grunt, Gulp, etc.)
+***
+
+Now that we have the targeted style library imported into our application, we can go ahead and import the components as it shows in the following code:
+
+  ```js
+  import {WebexAvatar} from '@webex/components';
+  import './App.css';
+
+  function App() {
+    return (
+      <div className='App'>
+        <WebexAvatar title='Webex Avatar' />
+      </div>
+    );
+  }
+
+  export default App;
+  ```
+
+*Happy Coding!*
+
+
+<a name='contributions'></a>
+## Contributions
+As much as we truly appreciate your contributions to this projects, there are some rules that we expect all of our contributors to follow.
+
+### Issues
+Please open an [issue](https://github.com/webex/components/issues) and we will get to it in an orderly manner. Please leave as much as information as possible for a better underestanding.
+### Coding
+We are using [webex standard eslint](https://github.com/webex/web-styleguide) and [husky](https://github.com/typicode/husky) to lintify the code style. Please make sure to prettify your code before committing.
+
+  ```bash
+  npm run prettier:write
+  ```
+
+### Release Process
+We are using [semantic-release](https://github.com/semantic-release/semantic-release) to fully automate the version management and package publishing. By default `semantic-release` uses the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) and triggers releases based on the following rules:
+
+<center>
+
+|Commit|Release type|
+|------|------------|
+|Commit with breaking change| Major release|
+|Commit with type `feat`| Minor release|
+|Commit with type `fix` | Patch release|
+|Commit with type `perf`| Patch release|
+</center>
+
+#### Commit linter
+We are using [Commitlint](https://github.com/conventional-changelog/commitlint) and [husky](https://github.com/typicode/husky) to lintify the commit messages. Please make sure to choose the appropriate commit `type`, `scope` and `description`. More information [here](https://github.com/webex/components/blob/master/CONTRIBUTING.md).
+
+<a name='license'></a>
+## License and Support
+
+*MIT* License.
+
+### Support
+Please click [here](https://developer.webex.com/support) to choose the appropriate method for more help.
```

Comment:
Explicitly state "here" as the developer portal.
`
For more developer resources, tutorials and support, visit the Webex developer portal, https://developer.webex.com
`